cluster/CHAPTER.rst

```diff
@@ -1269,11 +1248,9 @@ When something *does* break, all of the debugging support becomes critical: if t
 
 I performed some manual fuzz testing of cluster during development, but a full fuzz-testing infrastructure is beyond the scope of this project.
 
-Implementation Challenges
-=========================
+## Implementation Challenges
 
-Follow the Leader
```

Comment:
Hah, good catch.  I think I had other challenges in mind (perhaps never written down). Let's leave it here, and just rename it to something colorful like "Power Struggles".  I think it fits in nicely between the more mundane testing stuff and the as-yet-unsolved "Further Extensions" section.